driver_cpl/shr/seq_flds_mod.F90

```diff
@@ -1335,6 +1335,15 @@ subroutine seq_flds_set(nmlfile, ID)
      attname  = 'Si_ifrac'
      call metadata_set(attname, longname, stdname, units)
 
+     ! Sea ice basal pressure
+     call seq_flds_add(i2x_states,"Si_bpress")
```

Comment:
It should not need an ifdef. The sea ice components are free not to fill in this field - and the ocean is free not to read it. So the only thing is a bit of computational expense for redistribution (since there is no regridding). I think a comment in the code specifying that this only applies to acme would be helpful.